webrtc.html

```diff
@@ -9303,7 +9348,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         "RTCDataChannelState" class="simple">
           <tbody>
             <tr>
-              <th colspan="2"><dfn>RTCDataChannelState</dfn> Enumeration description</th>
```

Comment:
I put the IDL definitions `RTCDataChannelPriority` and `RTCDataChannelState` in new sub-sections. So the new dfn is defined in the new section header few lines above.